### DISCUSSION TITLE
katex not rendered in <div></div> block and <p></p> paragraph

### DISCUSSION THREAD

**Discussion by ChiMaoShuPhy** [marp-team/marp#152] [Q&A] [answered]
Hello, I'm wondering is there a way to make math equation being rendered inside `<div></div>` or `<p></p>`, e.g,

```html
<div>
    $\int dx$
</div>
```

```html
<p>
    $\int dx$
</p>
```

Many thanks!

**Comment by yhatt** [comment, marked as answer]
If you want to wrap with HTML tags, you have to put blank lines around Markdown contents, to prevent parsing contents as a part of the HTML block.

```markdown
<div>

$\int dx$

</div>
```

Any Markdown syntaxes including math would never be parsed in the HTML block.  See CommonMark specification: https://spec.commonmark.org/0.30/#example-188 and https://spec.commonmark.org/0.30/#example-189

**Reply by ChiMaoShuPhy** [reply]
Now it works! :smiley:  Many thanks for your help!
